library/oraclelinux

```diff
@@ -25,7 +25,11 @@ Tags: 8-slim
 Architectures: amd64, arm64v8
 Directory: 8-slim
 
-Tags: 7.8, 7
+Tags: 7.9, 7
+Architectures: amd64, arm64v8
+Directory: 7.9
+
+Tags: 7.8
 Architectures: amd64, arm64v8
 Directory: 7.8
```

Comment:
I think you probably know what I'm going to say here... :smirk:

Any particular reason to keep this version listed here?  Removing tags here will remove them from the "Supported" section on the Hub readme (and will prevent us from spending cycles rebuilding them on the official build servers), but the tags will still be available to users who want them.  (See https://github.com/docker-library/official-images#library-definition-files for more detail on this.)

Reply:
(I suppose the same goes for 8.0 and 8.1, which we've likely talked about previously?)